modules/osd-create-cluster-ccs-aws.adoc

```diff
@@ -0,0 +1,227 @@
+. Click *Next*.
+. If you opted to install the cluster in an existing AWS VPC, provide your *Virtual Private Cloud (VPC) subnet settings* and select *Next*.
+You must have created the Cloud network address translation (NAT) and a Cloud router. See the "Additional resources" section for information about Cloud NATs and Google VPCs.
++
+[NOTE]
+====
+You must ensure that your VPC is configured with a public and a private subnet for each availability zone that you want the cluster installed into. If you opted to use PrivateLink, only private subnets are required.
+====
+.. Optional: Expand *Additional security groups* and select additional custom security groups to apply to nodes in the machine pools that are created by default. You must have already created the security groups and associated them with the VPC that you selected for this cluster. You cannot add or edit security groups to the default machine pools after you create the cluster.
++
+By default, the security groups you specify are added for all node types. Clear the *Apply the same security groups to all node types* checkbox to apply different security groups for each node type.
++
+For more information, see the requirements for _Security groups_ under _Additional resources_.
+. If you opted to configure a cluster-wide proxy, provide your proxy configuration details on the *Cluster-wide proxy* page:
++
+.. Enter a value in at least one of the following fields:
+** Specify a valid *HTTP proxy URL*.
+** Specify a valid *HTTPS proxy URL*.
+** In the *Additional trust bundle* field, provide a PEM encoded X.509 certificate bundle. The bundle is added to the trusted certificate store for the cluster nodes. An additional trust bundle file is required if you use a TLS-inspecting proxy unless the identity certificate for the proxy is signed by an authority from the {op-system-first} trust bundle. This requirement applies regardless of whether the proxy is transparent or requires explicit configuration using the `http-proxy` and `https-proxy` arguments.
++
+.. Click *Next*.
++
+For more information about configuring a proxy with {product-title}, see _Configuring a cluster-wide proxy_.
+
+. In the *CIDR ranges* dialog, configure custom classless inter-domain routing (CIDR) ranges or use the defaults that are provided.
++
+[NOTE]
+====
+If you are installing into a VPC, the *Machine CIDR* range must match the VPC subnets.
+====
++
+[IMPORTANT]
+====
+CIDR configurations cannot be changed later. Confirm your selections with your network administrator before proceeding.
+====
++
+. Accept the default application ingress settings, or to create your own custom settings, select *Custom Settings*.
+
+.. Optional: Provide route selector.
+.. Optional: Provide excluded namespaces.
+.. Select a namespace ownership policy.
+.. Select a wildcard policy.
++
+For more information about custom application ingress settings, click the information icon provided for each setting.
```

Comment:
All these sections in wizard are part of Networking configuration step i.e. part of step 12 in this doc. We would need to move these definition there.